app/test_main.py

```diff
@@ -1,3 +1,30 @@
 from app.main import get_human_age
 
-# write your code here
+def test_get_human_age_zero_ages():
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repetition and make the tests more concise and maintainable, as suggested in the checklist.